Formula/ssh-agent-filter.rb

```diff
@@ -0,0 +1,61 @@
+class SshAgentFilter < Formula
+  desc "filtering proxy for ssh-agent"
+  homepage "https://github.com/tiwe-de/ssh-agent-filter"
+  url "https://github.com/tiwe-de/ssh-agent-filter/archive/0.4.2.tar.gz"
+  sha256 "5586e40da34f4afd873d7ff30f115c4a1b9455c1b81f3c4679c995db7615b080"
+
+  depends_on "help2man" => :build
+  depends_on "boost"
+  depends_on "nettle"
+
+  patch do
+    # upstream didn't want this because realpath isn't included in coreutils for Debian wheezy.
+    # see https://github.com/tiwe-de/ssh-agent-filter/pull/5
+    url "https://patch-diff.githubusercontent.com/raw/tiwe-de/ssh-agent-filter/pull/5.diff"
+    sha256 "307ccd6e872a3f3b999d8a4368d152dfbd3a7e43b4cff0cacbc0390c78b2976b"
+  end
+
+  patch do
+    # https://github.com/tiwe-de/ssh-agent-filter/pull/6
+    url "https://patch-diff.githubusercontent.com/raw/tiwe-de/ssh-agent-filter/pull/6.diff"
+    sha256 "5bd81db7cbfaf04e681f343ef6742061eaf6c9ead7d30c3fe6bf98eb8690c32f"
+  end
+
+  patch :DATA
```

Comment:
Actually the manpage contained in the bottles generated by jenkins have the correct umlaut.
Can you test if the output manpage changes with the Locale::gettext perl module installed?